src/complete-data/state-array.js

```diff
@@ -0,0 +1,54 @@
+export const stateArray = {
```

Comment:
Does it matter that this is an object (rather than an array)?

This is 100% nitpicky :)